core/network/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    kotlin("jvm")
+}
+
+group = "com.boswelja"
+version = "1.0-SNAPSHOT"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:1.6.4")
```

Comment:
Can we just use `kotlinx-coroutines-core` here and let it pull the correct libs?

Reply:
I don't think so. Dispatchers won't be available unless I'm missing something I could not use them without this import

Reply:
Ah right, I forgot IO dispatcher is only JVM
Carry on 😄